### ISSUE TITLE
Loading `chrome://accessibility` is broken

### ISSUE THREAD

**Issue by codebytere** [electron/electron#47496]
### Electron Version

38.0.0-nightly.20250602

### What operating system(s) are you using?

macOS

### Operating System Version

macOS 15.5

### What arch are you using?

arm64

### Last Known Working Electron version

N/A

### Expected Behavior

`chrome://accessibility` should load without issue.

### Actual Behavior

The page loads but is missing several components, and the following error appears in console:

```
[51206:0618/080423.037362:ERROR:CONSOLE:4] "Uncaught TypeError: Cannot read properties of undefined (reading 'native')", source: chrome://accessibility/accessibility.js (4)
```
